GVFS/GVFS.FunctionalTests/Tools/GVFSHelpers.cs

```diff
@@ -179,6 +183,24 @@ public static string GetInternalParameter(string maintenanceJob = "null", string
                     $"\\\"PackfileMaintenanceBatchSize\\\":{packfileMaintenanceBatchSize}}}\"";
         }
 
+        public static void HandleRegisterForOfflineIO()
+        {
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
+            {
+                uint result = RegisterForOfflineIO();
+                result.ShouldEqual<uint>(PrjFSResultSuccess, $"{nameof(HandleRegisterForOfflineIO)} failed (result = {result})");
+            }
+        }
+
+        public static void HandleUnregisterForOfflineIO()

```

Comment:
Naming suggestion for `HandleRegisterForOfflineIO` and `HandleUnregisterForOfflineIO`.  

The name of these made me think they might be callback handlers.  I think you could leave them simply as `RegisterForOfflineIO` and `UnregisterForOfflineIO` and on non-OSX platforms the methods no-op.